src/pages/Patch.tsx

```diff
@@ -61,3 +83,11 @@ export const Patch = () => {
     </PageWrapper>
   );
 };
+
+export const Patch = () => {
+  return (
+    <BannerContextProvider>
+      <PatchCore />
+    </BannerContextProvider>
+  );
+};
```

Comment:
this is not ideal but i could not successfully get other approaches to work. i tried to automatically render each route wrapped in `BannerContextProvider` where all routes are rendered. Spent about an hour on that but it was not successful.

Reply:
I'm not sure if you tried this but another abstraction to clean this up could be to create a higher order component that returns the argument wrapped with BannerContextProvider. So it could look something like `export const Patch = addBanner(() => {...})`

Reply:
The higher order component idea is great. Wrapping each route with a `withBannerContext` HOC would work great. And allows us to declare which routes have the context and saves some repetition from declaring it in each page.

Reply:
such a good idea!